### PR TITLE
Updated metadata and paper organization

### PR DESCRIPTION
**Pull request by varungohil**
Linking the issue #5 

Hi, we compiled these files on overleaf as we could not get make to work. 
Can you please check if files are compiling properly with make?

Thanks

### CODE REVIEW

File: gohilReOneTicket/journal/metadata.tex [hunk 1]
```diff
@@ -37,7 +37,8 @@
 \def \authorsSHORT{Gohil, Narayanan and Jain}
 \title{\articleTITLE}
 \date{}
-\author[1]{Varun Gohil}
-\author[1,]{S. Deepak Narayanan}
-\author[1,]{Atishay Jain}
-\affil[1]{Indian Institute of Technology, Gandhinagar, Gandhinagar, India}
+\author[1$\dagger$]{Varun Gohil}
```
Comment (koustuvsinha) [inline gohilReOneTicket/journal/metadata.tex]: compilation fails here. there should be a comma between 1 and `$\dagger$`

File: gohilReOneTicket/journal/metadata.tex [hunk 2]
```diff
@@ -37,7 +37,8 @@
...
+\author[1$\dagger$]{S. Deepak Narayanan}
+\author[1$\dagger$]{, Atishay Jain}
+\affil[1]{Indian Institute of Technology Gandhinagar, Gandhinagar, India}
+\affil[$\dagger$]{Equal Contribution}   2
```
Comment (koustuvsinha) [inline gohilReOneTicket/journal/metadata.tex]: Remove the trailing 2

File: gohilReOneTicket/journal/metadata.tex (hunk 1 above)
Comment (varungohil) [reply gohilReOneTicket/journal/metadata.tex]: Done

File: gohilReOneTicket/journal/metadata.tex (hunk 2 above)
Comment (varungohil) [reply gohilReOneTicket/journal/metadata.tex]: Done

**Pull request merged**